.circleci/config.yml

```diff
@@ -87,6 +100,28 @@ jobs:
           command: echo 'export FORMATTED_CIRCLE_BRANCH=$(echo ${CIRCLE_BRANCH} | sed "s/\//-/g")' >> $BASH_ENV
       - deploy-to-aws:
           subdirectory: '/dev/${FORMATTED_CIRCLE_BRANCH}'
+  # deploys support branch assets to an uncached folder in the S3 bucket named by branch
+  deploy_support:
```

Comment:
what is different about deploy_support and deploy_feature_i18n compared to deploy_branch?

Reply:
Oh, I can consolidate these

Reply:
Actually I can consolidate deploy_support and deploy_feature_i18n, but their difference to deploy_branch is that deploy_branch doesn't depend on build_all_locales